### PR TITLE
Add 'continue' command

### PR DESCRIPTION
**Pull request by filipesaraiva**

### CODE REVIEW

**Comment by filipesaraiva** [comment]
Hello @rbrito, after you accept this pull request I will open the pull request for 'break' command.

File: algorithms.dtx [hunk 1]
```diff
@@ -746,6 +746,7 @@ USA.
 % \newcommand{\algorithmicendrepeat}{\algorithmicend\ \algorithmicrepeat}
 % \newcommand{\algorithmicuntil}{\textbf{until}}
 % \newcommand{\algorithmicprint}{\textbf{print}}
+% \newcommand{\algorithmiccontinue}{\textbf{continue}}
```
Comment (rbrito) [inline algorithms.dtx:763]: Thanks for your contribution, @filipesaraiva. Please, add a few lines describing the use of continue... Probably, the best place for this is in the for or in the while section of the document...

**Review by rbrito — CHANGES_REQUESTED** [review]
After you send the documentation for continue, I will merge your pull request and try to get this released as soon as possible in CTAN.

File: algorithms.dtx (hunk 1 above)
Comment (filipesaraiva) [reply algorithms.dtx:763]: Thanks @rbrito, I pushed more one commit adding documentation to `continue`. This pull request was updated.

File: algorithms.dtx [hunk 2]
```diff
@@ -376,6 +376,20 @@ USA.
...
+% commands are present in most of programming languages and can be used for
+% algorithm description.
+%
+% \emph{continue} is used to jump to the next iteration of the loop. All the
+% commands after \emph{continue} will not run for the current loop cycle.
+%
+% \emph{continue} can be used with the follow form:
+% \begin{verbatim}
+% \CONTINUE
+% \end{verbatim}
```
Comment (rbrito) [inline algorithms.dtx:391]: Excellent!

**Comment by rbrito** [comment]
Thanks a lot.

**Pull request merged**